vispy/scene/systems.py

```diff
@@ -35,6 +38,7 @@ def process(self, event, node):
                 self.process(event, sub_node)
             finally:
                 event.pop_node()
+            prof('process child %s', sub_node)
```

Comment:
No--just like with logging, we avoid string formatting until the message is displayed.